skada/deep/_adversarial.py

```diff
@@ -372,18 +372,49 @@ def CDAN(
 
 
 class ModifiedCrossEntropyLoss(torch.nn.Module):
-    """Modified CrossEntropyLoss.
-    Implements modified CrossEntropyLoss as described in (29) from [35]_.
+    """Modified CrossEntropyLoss as described in (29) from [35]_ with label smoothing.
+
+    Parameters
+    ----------
+    smoothing : float, default=0.1
+        Smoothing factor for label smoothing.
     """
 
-    def __init__(self):
+    def __init__(self, smoothing=0.1):
```

Comment:
non 0 default? it that used in the paper?

Reply:
they use 0 but I guess we want something robust and stable by default?

Reply:
log(1-prob) is likely to never work